src/main/webapp/js/components/interface/tutorial/Tutorial.less

```diff
@@ -29,7 +29,6 @@
 	width: auto;
 	minheight: 0px;
 	maxheight: none;
-	height: 80%;
```

Comment:
@mattearnshaw this makes the arrows move :)

Reply:
this is what I was trying to fix, as you can see there is cut-off and you can't scroll either. I'll try and make it so button are fixed and scroll works (without breaking other deployments)

![tutorial](https://user-images.githubusercontent.com/679927/33620368-9134df58-d9df-11e7-8deb-935244e4f013.png)

Reply:
Where is the code for this particular screenshot stored? Nothing to do with this issue, but the equation for g_K is slightly odd: the maximum conductance for K is weirdly typeset as a product of _g_ and max_K (whatever that is), while it should read as a single parameter - usually typeset as `\bar{g}_K` 
![image](https://user-images.githubusercontent.com/3452783/33621379-ab61a4d2-d9d1-11e7-8e54-b02710b335aa.png?v=4&s=2)

Reply:
@borismarin https://github.com/tarelli/tutorials/blob/182d2952ab56440a6ef7bd45400550c3ec46fecf/1_hh_practical/eq1.png

Reply:
@borismarin Equations were lifted from @pgleeson doc btw there's probably a better way to visualizing them other than embedding pre-rendered images.